framework/src/consumption/lib/quicksight/quicksight-subscription-props.ts

```diff
@@ -0,0 +1,91 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Duration, RemovalPolicy } from 'aws-cdk-lib';
+
+
+/**
+ * The properties for the `QuickSightSubscription` construct
+ */
+
+export interface QuickSightSubscriptionProps {
+
+  /**
+   * The name of your Amazon QuickSight account.
+   * This name is unique over all of Amazon Web Services, and it appears only when users sign in.
+   * You can't change AccountName value after the Amazon QuickSight account is created.
+   */
+  readonly accountName: string;
+
+  /**
+   * The email address that you want Amazon QuickSight to send notifications to regarding your Amazon QuickSight account or Amazon QuickSight subscription.
+   */
+  readonly notificationEmail: string;
+
+  /**
+  * The edition of Amazon QuickSight that you want your account to have. Currently, you can choose from ENTERPRISE or ENTERPRISE_AND_Q .
+  *  @default - ENTERPRISE is used as default.
+  */
+  readonly edition: QuickSightEdition;
+
+  /**
+   * The Amazon Web Services account ID of the account that you're using to create your Amazon QuickSight account.
+   */
+  readonly awsAccountId: string;
```

Comment:
what's the benefit of configuring a different account than the CDK one?

Reply:
mm, this could be benefitial to support the creation of pipelines if we want to support them in the future: